### PR TITLE
Available rides screen added

### PR DESCRIPTION
**Pull request by mananssh**
Issue fixed: #4 
Changes in this PR:
- Added assets for search and filter button icons
- Added json file for dummy rides data
- Created available rides screen
- Changed name format in ride card to be marquee text

Screenshot of available rides screen:
WhatsApp Image 2024-01-06 at 21 57 00_6598943d

### CODE REVIEW

File: components/rideCard.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import AppColors from '../design-system/colors';
+import MarqueeText from 'react-native-marquee';
 
 interface CardVariables {
   data: {
     startingLocation: string;
     destination: string;
-    date: Date;
-    time: Date;
+    date: any;
```
Comment (RohanKhatua) [inline components/rideCard.tsx]: Why any? this makes it not typesafe

**Review by RohanKhatua — CHANGES_REQUESTED** [review]
Please see review comments - RK

File: data/availableRides.json
Comment (anandrajaram21) [inline data/availableRides.json:1]: i'm assuming this file will be removed later right? Once the connection to the backend is done

**Review by anandrajaram21 — CHANGES_REQUESTED** [review]
can you please resolve merge conflicts and the comment

File: data/availableRides.json
Comment (mananssh) [reply data/availableRides.json:1]: yes, all the files in this dir are dummy data for testing.

**Comment by mananssh** [comment]
@anandrajaram21 @RohanKhatua sorry for the delay, necessary changes have been made

**Review by RohanKhatua — APPROVED** [review]
LGTM YOLO

**Pull request merged**